app/src/assets/i18n/en.json

```diff
@@ -269,6 +269,9 @@
                 "MESSAGE": "Leaving will remove your access.",
                 "YES_LABEL": "OK",
                 "NO_LABEL": "Cancel"
+            },
+            "OPEN_IN_APP": {
+              "TITLE": "Open in APP"
```

Comment:
Needs some work on the wording, "open in APP" (why uppercase?) makes no sense... maybe should be saying about mobile app, or something similar

Reply:
noted and changed it to the desired.